source/dem/adaptive_sparse_contacts.cc

```diff
@@ -165,8 +165,22 @@ DisableContacts<dim>::identify_mobility_status(
                            mpi_communicator);
   mobility_at_nodes = 0;
 
+  // If the advection of particles setting is enabled (useful for CFD-DEM),
+  // mobility status are different: inactive and static_active status are
+  // advected and advected_active since they are not handled in the same way as
+  // in DEM only. They have a special handling in the velocity integration step
+  // because cell averaged velocity and acceleration are applied to particles,
+  // so they need this different status. The criteria for those status are the
+  // same as for DEM.
+  mobility_status inactive_status = (!advect_particles_enabled) ?
+                                      mobility_status::inactive :
+                                      mobility_status::advected;
+  mobility_status active_status   = (!advect_particles_enabled) ?
+                                      mobility_status::static_active :
+                                      mobility_status::advected_active;
+
   // Check if the cell is empty (n_particle = 0), if so, nodes and cells are
-  // flagged as empty mobility status (3)
+  // flagged as empty mobility status (5)
```

Comment:
Ideally if this is an enum or an enum class, the number should really not matter. Not sure it's worth metnioniong (well maybe becuase you use it in the post-processing?

Reply:
I added it so it is easier to understand, but maybe that's not relevant